### DISCUSSION TITLE
More flexible filters on the calendar

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Planner#1271] [Feature Requests]
Spoke with a customer today. They've love the ability to choose multiple statuses on the Calendar:

![screen shot 2018-09-18 at 9 29 12 am](https://user-images.githubusercontent.com/3868028/45690860-517b0180-bb25-11e8-8988-9192b3237d1e.png)

**Comment by stevejburge** [comment]
Here's another filter request from a user:

> search posts from within the calendar – have to scroll through the dates manually and look for them one by one

**Comment by stevejburge** [comment]
One more user:

> I want to search post by tag in Calendar

![unnamed-2](https://user-images.githubusercontent.com/3868028/77164134-ff060280-6a85-11ea-840a-ea53b4305e3f.png)

**Comment by stevejburge** [comment]
One more request for this today, including the options to:
- Choose post types
- Choose from other taxonomies

**Comment by stevejburge** [comment]
Request: hide some options in the filters:

>  We would want to use the Editorial Calendar but we wouldn't want the calendar to show all of the categories that have been published as some of them are not important to editors. Would there be any way of filtering the categories in the Editorial Calendar so that certain categories would never be displayed? This would be a very important feature for us.

**Comment by andergmartins** [comment]
Maybe we could give the option to save presets or different views for the calendar? Those views could be set for specific user roles, or be available to anyone. You could basically configure the filters like you want, click on "Save view", give it a name and use it as default or select it when ever you want.

Something like this: https://docs.clickup.com/en/articles/3007059-saving-view-changes

A select field (or other UI element) could be used to select the view you want.

**Comment by stevejburge** [comment]
Thanks @andergmartins 

Yes, great idea. That would solve this other request https://github.com/publishpress/PublishPress/issues/458
